### DISCUSSION TITLE
FERC714 hourly data timestamp right-side bin labeling

### DISCUSSION THREAD

**Discussion by dshen109** [catalyst-cooperative/pudl#3924] [Help Me!]
I am using some of the FERC 714 hourly data and wanted to confirm if PUDL is using labels corresponding to the right-side of the hourly interval (e.g. a label of 2024-01-02 01:00 corresponds to aggregating measurements from 2024-01-02 00:00:01 to 2024-01-02 00:01:00)?

Btw thanks for this great application!

**Comment by zaneselvans** [comment]
I think different hourly data sources that we've integrated have different conventions as to whether they're hour-beginning or hour-ending (We should really standardize them all to the same convention!)

If I recall correctly the FERC-714 data is hour-ending, so 2024-01-02 01:00 would correspond to the hour between midnight and 1am.

**Reply by zaneselvans** [reply]
(but also, the respondents are not super consistent in their reporting so there could definitely be some variability between years, and between different respondents)
